src/components/RelatedVideo.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const RelatedVideo = ({className, url, picture, title, description, showRelatedVideo}) => {
+    return (
+        <>
+            <div className={className} onClick={() => showRelatedVideo(url, title, description)}>
+                <div>
+                    <img src={picture} alt="No picture" height="95" width="130" />
+                </div>
+                <div style={{display: 'flex', flexDirection: 'column', justifyContent: 'center'}}>
```

Comment:
try to use styled component instead of put in line styles (style={{display: 'flex', flexDirection: 'column', justifyContent: 'center'}})